<def f='include/boost/exception/exception.hpp' l='425' type='void boost::exception_detail::copy_boost_exception(void * , const void * )'/>
